### PR TITLE
Create 314-tribute-to-talk.md

### PR DESCRIPTION
**Pull request by rachelhamlin**

### CODE REVIEW

File: ideas/314-tribute-to-talk.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+TtT Swarm
```
Comment (oskarth) [inline ideas/314-tribute-to-talk.md]: Missing metadata

File: ideas/314-tribute-to-talk.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+It allows individuals to monetize their time and attention by setting minimum required deposits to message them.
+
+## Contributors
+
+- Eric 
+- Vitaliy 
+- Maciej
+- Tetiana 
+
```
Comment (oskarth) [inline ideas/314-tribute-to-talk.md]: Smart contract dev? @richard-ramos ?

File: ideas/314-tribute-to-talk.md (hunk 1 above)
Comment (rachelhamlin) [reply ideas/314-tribute-to-talk.md]: Fixed. Spoke with Nabil and he'll handle budget post research phase.

File: ideas/314-tribute-to-talk.md (hunk 2 above)
Comment (rachelhamlin) [reply ideas/314-tribute-to-talk.md]: Most likely, yes. Shouldn't require too much of his time as the contract is already written but still eligible for changes. Working out his time with Iuri.

**Pull request merged**